packages/zoe/src/contracts/priceAggregator.js

```diff
@@ -513,6 +570,7 @@ const start = async (zcf, privateArgs) => {
        */
       record.querier = async timestamp => {
         // Submit the query.
+        /** @type {ParsableNumber} */
         const result = await E(oracle).query(query);
```

Comment:
Do we have reason to rely on the oracle to return a `ParseableNumber` without validating it?
If not, perhaps `@type {unknown}` is more appropriate to be sure we validate it before relying on its type.

I'm struggling to keep track of the external surface of this contract. Factoring out just that part and using patterns seems like it would be nice.

Reply:
> Do we have reason to rely on the oracle to return a ParseableNumber without validating it?

Good point. I added an assertion instead of an unchecked cast.